apps/www/registry/new-york/ui/command.tsx

```diff
@@ -5,8 +5,10 @@ import { type DialogProps } from "@radix-ui/react-dialog"
 import { MagnifyingGlassIcon } from "@radix-ui/react-icons"
 import { Command as CommandPrimitive } from "cmdk"
 
+
 import { cn } from "@/lib/utils"
-import { Dialog, DialogContent } from "@/registry/new-york/ui/dialog"
+import { VisuallyHidden } from "@radix-ui/react-visually-hidden"
+import { Dialog, DialogContent, DialogDescription, DialogTitle } from "@/registry/new-york/ui/dialog"
```

Comment:
The extra imports need to be removed
```suggestion
import { Dialog, DialogContent, DialogTitle } from "@/registry/new-york/ui/dialog"
```